packages/runtime/container-runtime/src/summary/summarizerNode/summarizerNode.ts

```diff
@@ -192,6 +192,46 @@ export class SummarizerNode implements IRootSummarizerNode {
 		return { summary: result.summary, stats: result.stats };
 	}
 
+	public validateSummary() {
+		this.validateSummaryCore(false);
```

Comment:
Nit: You could just do `public validateSummary(parentSkipRecursion: boolean = false)` instead of adding the `Core` version

Reply:
I just kept it similar to other functions where there is a non-core function for the "root" node which calls the core version for all other child nodes.